test/recipes/80-test_cms.t

```diff
@@ -1239,6 +1239,22 @@ with({ exit_checker => sub { return shift == 6; } },
             "Check failure during BIO setup with -stream is handled correctly");
     });
 
+# Check that kari encryption with originator does not segfault
+with({ exit_checker => sub { return shift == 3; } },
+    sub {
+        SKIP: {
+          skip "EC is not supported in this build", 1 if $no_ec;
+
+          ok(run(app(['openssl', 'cms', '-encrypt',
+                      '-in', srctop_file("test", "smcont.txt"), '-aes128',
+                      '-recip', catfile($smdir, "smec1.pem"),
+                      '-originator', catfile($smdir, "smec3-cert.pem"),
+                      '-inkey', catfile($smdir, "smec3-key.pem")
+                    ])),
+              "Check kari encyryption with originator normal failure");
```

Comment:
```suggestion
              "So far, kari encyryption with static originator is not supported");
```

Reply:
Used slightly different workding but the meaning should be as suggested.

Reply:
The text you wrote "Check failure for currently not supported encryption with static originator"
misses the 'key agreement' part. It sounds like all encryption with static originator is not supported.

Reply:
There is a leftover extra empty line 1255.